code/game/objects/items/stacks/medical.dm

```diff
@@ -163,13 +136,19 @@
 	icon_state = "ointment"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_burn = 40
+	var/heal_burn = 25
 	self_delay = 20
+	max_amount = 12
 	grind_results = list("silver_sulfadiazine" = 10)
 
+/obj/item/stack/medical/ointment/heal(mob/living/M, mob/user)
+	if(M.stat == DEAD)
+		to_chat(user, "<span class='notice'> [M] is dead. You can not help [M.p_them()]!</span>")
+		return
+	if(iscarbon(M))
+		return heal_carbon(M, user, 0, heal_burn)
+	to_chat(user, "<span class='notice'>You can't heal [M] with the \the [src]!</span>")
+
 /obj/item/stack/medical/ointment/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is squeezing \the [src] into [user.p_their()] mouth! [user.p_do(TRUE)]n't [user.p_they()] know that stuff is toxic?</span>")
 	return TOXLOSS
-
-/obj/item/stack/medical/get_belt_overlay()
-	return mutable_appearance('icons/obj/clothing/belt_overlays.dmi', "pouch")

```

Comment:
Code is fine and dandy, and I do not mind the healing power / uses trade. Just bring back these couple lines before I merge it, ok?